useraudit/middleware.py

```diff
@@ -1,11 +1,12 @@
 import threading
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
This will fail on pre 1.10 Django.

you should do something along these lines:

https://github.com/muccg/django-iprestrict/blob/4db839c123ae90afb82438cb500b5f3d54477ea9/iprestrict/middleware.py#L10